packages/wallet-sdk/src/models/profile.ts

```diff
@@ -88,7 +88,7 @@ export const fetchAccountProfileUrl = async ({
 };
 
 export function signProfileForUpload({ profile, account }: { profile: Profile; account: Account }) {
-  const privateKey = account.dataPrivateKey;
+  const privateKey = account.stxPrivateKey.slice(0, 64);
```

Comment:
What is the relevance of 64 here?
```suggestion
  const descriptiveNameOfWhatNumberIs = 64
  const privateKey = account.stxPrivateKey.slice(0, descriptiveNameOfWhatNumberIs);
```

Reply:
@kyranjamie Is the part required for compressed public key. 
There are 8 places in the repo that uses this expression `slice(0, 64)`. Should I replace all of them? Might be better as a separate PR.

Reply:
if i'm not mistaken, it's the difference between a compressed and uncompressed key. compressed keys have extra bytes at the end that show it's compressed, typically it's `01`. this removes that extra byte. uncompressed keys don't have it, so they are already at length 64. as @friedger said, it's used all over in the repo, but could be made into a helper function that shows why this is done. 

@zone117x could probably correct me if i'm wrong